maintenance/README.md

```diff
@@ -0,0 +1,33 @@
+# Maintenance mode
+
+Enable or disable maintenance mode for a service
+
+## Inputs
+- `environment`: Name of the app environment affected (Required)
+- `azure-credentials`: A JSON string containing service principle credentials (Required)
+- `mode`: Mode to set, either enable or disable (Required)
+- `docker-repository`: full name of the docker repository for the maintenance image (Required)
+- `github-token`: github token that can push the maintenance image to the docker repository (Required)
+
+## Example
+
+```yaml
+jobs:
```

Comment:
Could we add an example of worfkow dispatch with the choice input? This may help understanding the `${{ inputs.mode }}` below

Reply:
added inputs to readme